projects/wave-core-new/src/lib/operators/types/expression-type.model.ts

```diff
@@ -0,0 +1,107 @@
+import {OperatorType, OperatorTypeDict, OperatorTypeMappingDict} from '../operator-type.model';
+
+import {Unit, UnitDict, UnitMappingDict} from '../unit.model';
+import {DataType, DataTypes} from '../datatype.model';
+
+/**
+ * The expression specifies calculations between the rasters.
+ * There must be a valid data type and unit for the resulting raster.
+ */
+interface ExpressionTypeConfig {
+    expression: string;
+    datatype: DataType;
+    unit: Unit;
+}
+
+interface ExpressionTypeMappingDict extends OperatorTypeMappingDict {
+    expression: string;
+    datatype: string;
+    unit: UnitMappingDict;
+}
+
+export interface ExpressionTypeDict extends OperatorTypeDict {
+    expression: string;
+    datatype: string;
+    unit: UnitDict;
+}
```

Comment:
There is no mapping anymore and these dicts should now fall into one since we store it in the server's format.

Reply:
1187fb5a6148657ae9eb1e960c54ed287d841f02